### PR TITLE
Add warning deletion screen for Antigen and PCR (EXPOSUREAPP-6032)

### PR DESCRIPTION
**Pull request by axelherbstreith**
Currently only added the new screen. Connection to the submission flow will be done in a separate PR.

image

### CODE REVIEW

**Review by SamuraiKek — DISMISSED** [review]
Looks good. Spacing and layout looks as in the design.

File: Corona-Warn-App/src/main/res/layout/fragment_submission_deletion_warning.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        app:layout_constraintBottom_toTopOf="@id/continue_button"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/toolbar">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <ImageView
```
Comment (AlexanderAlferov) [inline Corona-Warn-App/src/main/res/layout/fragment_submission_deletion_warning.xml:34]: As far as I know we should use androidx.appcompat.widget.AppCompatImageView

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -1973,5 +1990,6 @@
     <!-- XMIT: Trace location poster share -->
     <string name="trace_location_organiser_poster_share">"Teilen"</string>
     <!-- XHED: Trace location poster title -->
-    <string name="trace_location_organiser_poster_title">"Druckversion"</string>
+    <string name="trace_location_organiser_poster_title">"Druckversion"</string>F
```
Comment (AlexanderAlferov) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1993]: Miss-type at the end of the line 😅

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/deletionwarning/SubmissionDeletionWarningFragment.kt [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import de.rki.coronawarnapp.util.di.AutoInject
+import de.rki.coronawarnapp.util.ui.viewBindingLazy
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
+import de.rki.coronawarnapp.util.viewmodel.cwaViewModels
+import javax.inject.Inject
+
+class SubmissionDeletionWarningFragment : Fragment(R.layout.fragment_submission_deletion_warning), AutoInject {
+
+    @Inject lateinit var viewModelFactory: CWAViewModelFactoryProvider.Factory
+    private val vm: SubmissionDeletionWarningFragmentViewModel by cwaViewModels { viewModelFactory }
```
Comment (AlexanderAlferov) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/deletionwarning/SubmissionDeletionWarningFragment.kt:18]: Would be great to have it as viewModel (we are trying to streamline it through our application 🚀 )

**Review by AlexanderAlferov — CHANGES_REQUESTED** [review]
Just small changes - everything else looks nice 🚀

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -1367,6 +1367,23 @@
...
 
+
+    <!-- XBUT: submission deletion warning button continue -->
+    <string name="submission_deletion_warning_continue_button">"Weiter"</string>
+    <!-- XBUT: submission deletion warning button cancel -->
+    <string name="submission_deletion_warning_cancel_button">"Abbrechen"</string>
+    <!-- XHED: submission deletion warning title -->
+    <string name="submission_deletion_warning_title">"Hinweis"</string>
+    <!-- YTXT: Headline for rapid test submission deletion warning -->
+    <string name="submission_deletion_warning_headline_antigen_test">"Sie haben bereits einen Schnelltest registriert."</string>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1378]: Shouldn't "erfasst"/"registriert" be used consistently? Both are okay, although I think for scanning a QR code, "registrieren" is more precise, if a code is entered manually, "erfassen" is fine.

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -1367,6 +1367,23 @@
...
+    <!-- YTXT: Body for rapid test submission deletion warning -->
+    <string name="submission_deletion_warning_body_antigen_test">"Sie haben bereits einen Schnelltest erfasst. Die App kann maximal einen Schnelltest und einen PCR-Test gleichzeitig verwalten. Wenn Sie einen weiteren Schnelltest erfassen, wird der erste Schnelltest aus der App gelöscht."</string>
+    <!-- YTXT: Headline for PCR test submission deletion warning -->
+    <string name="submission_deletion_warning_headline_pcr_test">"Sie haben bereits einen PCR-Test registriert."</string>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:1382]: See other comment

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -1375,6 +1375,23 @@
...
 
+
+    <!-- XBUT: submission deletion warning button continue -->
+    <string name="submission_deletion_warning_continue_button">"Weiter"</string>
+    <!-- XBUT: submission deletion warning button cancel -->
+    <string name="submission_deletion_warning_cancel_button">"Abbrechen"</string>
+    <!-- XHED: submission deletion warning title -->
+    <string name="submission_deletion_warning_title">"Hinweis"</string>
+    <!-- YTXT: Headline for antigen test submission deletion warning -->
+    <string name="submission_deletion_warning_headline_antigen_test">"Sie haben bereits einen Schnelltest registriert."</string>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values/strings.xml:1386]: registriert vs. erfasst -> see comment above

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -1375,6 +1375,23 @@
...
+    <!-- YTXT: Body for antigen test submission deletion warning -->
+    <string name="submission_deletion_warning_body_antigen_test">"Sie haben bereits einen Schnelltest erfasst. Die App kann maximal einen Schnelltest und einen PCR-Test gleichzeitig verwalten. Wenn Sie einen weiteren Schnelltest erfassen, wird der erste Schnelltest aus der App gelöscht."</string>
+    <!-- YTXT: Headline for PCR test submission deletion warning -->
+    <string name="submission_deletion_warning_headline_pcr_test">"Sie haben bereits einen PCR-Test registriert."</string>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/res/values/strings.xml:1390]: registriert vs. erfasst -> see comment above

**Review by AlexanderAlferov — DISMISSED** [review]

**Review by SamuraiKek — DISMISSED** [review]

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 4 above)
Comment (vaubaehn) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:1378]: @SabineLoss 
In my understanding "Erfassen" is the process to enter the data into the app, either by scanning QR code or by entering manually. "Registrieren" describes the process, where the data is submitted from app to the backend, and a token is registered on the server and sent back to the app.
Just my 2 cents

**Comment by janetback** [comment]
Hi @axelherbstreith, @SabineLoss @vaubaehn, although @vaubaehn 's distinction is correct, this is a very technical distinction which I believe does not need to be made on the UI itself. @SabineLoss , you are right, we should stay consistent here. In most instances we have used 'registered', which is why I suggest changing to 'registrieren' in this case too. 
@axelherbstreith - I have already adapted Figma. Could you please change the strings file here too? Thanks.

**Review by janetback — APPROVED** [review]
UA approved

**Review by AlexanderAlferov — APPROVED** [review]

**Pull request merged**